Documentation/Concepts/Backend/SystemModules/Index.rst

```diff
@@ -1,77 +1,113 @@
+.. _system-modules-backend-users:
 
-.. index:: Modules; DB check
-.. _system-modules-dbcheck:
+Backend Users
+=============
 
-DB check
-========
+The module :guilabel:`System > Backend Users` is used to create, edit and delete
+backend users.
 
-.. important::
+See also TYPO3 Explained,
+:ref:`Backend user management <t3coreapi:user-management>`.
 
-   "DB check and :ref:`system-modules-configuration` are only available
-   if the system extension "lowlevel" was installed and activated.
+.. _system-modules-reactions:
 
-   For Composer based systems (see also :ref:`installing-extensions`):
+Reactions (optional)
+====================
 
-   .. code-block:: bash
+This module is only available if the system extension
+:composer:`typo3/cms-reactions` is installed. This extension handles **incoming**
+webhooks to TYPO3. It also provides a corresponding backend module to manage
+reaction records.
 
-      composer req typo3/cms-lowlevel
+It has its own manual:
+:ref:`TYPO3 Reactions <typo3/cms-reactions:start>`.
 
+.. _system-modules-webhooks :
 
-The *Database (DB) check* module offers four functions related
-to the database and its content.
+Webhooks (optional)
+===================
+
+This module is only available if the system extension
+:composer:`typo3/cms-webhooks` is installed. This extension handles **outgoing**
+webhooks to TYPO3. It also provides a corresponding backend module to manage
+webhook records in the TYPO3.
+
+Unfortunately this extension is not documented at the time of writing.
+
+.. _system-modules-scheduler:
+
+Scheduler (optional)
+====================
+
+This module is only available if the system extension
+:composer:`typo3/cms-scheduler` is installed.
+
+The Scheduler supports one-time or periodic execution of tasks that can be
+delivered by any extension. It has its own manual:
+:ref:`TYPO3 Scheduler <typo3/cms-scheduler:start>`.
+
+.. _system-modules-dbcheck:
+
+DB check (optional)
+===================
 
-.. include:: /Images/AutomaticScreenshots/Modules/DbCheckModule.rst.txt
+This module is only available if the system extension
+:composer:`typo3/cms-lowlevel` is installed.
+
+The *Database (DB) check* module offers functions related
+to the database and its content.
 
 Record Statistics
-  Shows a count of the various records in the database,
-  broken down by type for pages and content elements.
+    Shows a count of the various records in the database,
+    broken down by type for pages and content elements.
 
 Relations
-  Checks if certain relations are empty or broken, typically
-  used to check if files are being referenced.
+    Checks if certain relations are empty or broken, typically
+    used to check if files are being referenced.
 
 Search
-  A tool to search through the whole database. It offers an
-  advanced mode which is similar to a visual query builder.
+    A tool to search through the whole database. It offers an
+    advanced mode which is similar to a visual query builder.
```

Comment:
What is a visual query builder? I would add an explanation here.

Reply:
We can enhance descriptions in a follow-up